js/src/admin/components/ExtensionPage.js

```diff
@@ -178,39 +178,40 @@ export default class ExtensionPage extends Page {
       items.add('authors', [icon('fas fa-user'), <span>{authors.join(', ')}</span>]);
     }
 
+    const infoData = {};
+
+    if (this.extension.extra['flarum-extension'].info) {
```

Comment:
Not sure if we need this wrapper: it doesn't save that much performance, and it adds an extra layer of nesting

Reply:
It would need to be checked each time in the loop. Is that ok?

Reply:
You also need to add `source` to `this.infoItems`. But it's a question of looping over 6 items vs not looping over 6 items. And performance-wise, that's extremely trivial IMO